spec/lib/models/i18n_repository_spec.rb

```diff
@@ -104,7 +104,7 @@
   describe '#translation_status_high_enough?' do
     let(:code) { 'test-code' }
     let(:percent) { 100 }
-    let(:translations_statuses) { [ { 'code' => code, 'translated_progress' => percent } ] }
+    let(:translations_statuses) { [{ 'code' => code, 'translated_progress' => percent }] }
```

Comment:
Line is too long. [90/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.